components/atoms/text/text.js

```diff
@@ -11,6 +11,7 @@ import styled, { css } from 'styled-components'
 
 export const Text = ({
   align,
+  bold,
```

Comment:
Isn't it better to add a prop like `weight` and have values like `light, bold, semi-bold, ...` there based on our designs?